### PR TITLE
PropCore: added propInertia()

### PR DESCRIPTION
**Pull request by PTugaSantos**
Sets the directional inertia

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/prop.lua [hunk 1]
```diff
@@ -297,6 +297,15 @@ e2function void entity:propDrag( number drag )
 	end
 end
 
+e2function void entity:propInertia( vector inertia )
+	if not PropCore.ValidAction(self, this, "inertia") then return end
+	if Vector( inertia[1], inertia[2], inertia[3] ):IsZero() then return end
```
Comment (AbigailBuccaneer) [inline lua/entities/gmod_wire_expression2/core/custom/prop.lua]: What happens if you set the inertia to zero?

Comment (PTugaSantos) [inline lua/entities/gmod_wire_expression2/core/custom/prop.lua]: Entity gets removed

**Comment by shadowscion** [comment]
can you change it to setInertia (setMass)?

**Comment by PTugaSantos** [comment]
Originally it was setInertia(), the reason for propInertia() was to leave all with the same syntax of propcore extension. (propBreak, propDrag, propDraw, propShadow, propSetBuoyancy...)

**Pull request merged**